src/wp-includes/script-loader.php

```diff
@@ -1681,7 +1681,6 @@ function wp_default_styles( $styles ) {
 
 	$wp_edit_blocks_dependencies = array(
```

Comment:
I think this name is kind of confusing, We should clarify maybe with a comment that these styles are the one that impact the "content" and are injected in the iframe.